dotcom-rendering/src/components/MostViewedRight.tsx

```diff
@@ -29,47 +30,89 @@ interface Props {
 	stickToTop?: boolean;
 }
 
+interface MostViewedRightPayloadType {
+	tabs: FETrailTabType[];
+}
+
 export const MostViewedRight = ({
 	limitItems = 5,
 	stickToTop = false,
 }: Props) => {
+	const testVariant = useDeeplyReadTestVariant();
 	const endpointUrl =
-		'https://api.nextgen.guardianapps.co.uk/most-read-geo.json?dcr=true';
-	const { data, error } = useApi<FETrailTabType>(endpointUrl);
+		'https://api.nextgen.guardianapps.co.uk/most-read-with-deeply-read.json';
+	const { data, error } = useApi<MostViewedRightPayloadType>(endpointUrl);
 
 	if (error) {
-		window.guardian.modules.sentry.reportError(error, 'most-viewed-right');
+		window.guardian.modules.sentry.reportError(
+			error,
+			'most-viewed-deeply-read-right',
+		);
 		return null;
 	}
 
 	if (data) {
-		const trails: TrailType[] = data.trails
-			.slice(0, limitItems)
-			.map(decideTrail);
+		const mostReadTrails = data.tabs[0]?.trails
+			?.slice(0, limitItems)
+			?.map(decideTrail);
+		const deeplyReadTrails =
+			testVariant === 'deeply-read-and-most-viewed' ||
+			testVariant === 'deeply-read-only'
+				? data.tabs[1]?.trails?.slice(0, limitItems)?.map(decideTrail)
+				: undefined;
 
-		// Look I don't know why data-component is geo-most-popular either, but it is, ok? Ok.
 		return (
 			<div
 				css={[wrapperStyles, stickToTop && stickyStyles]}
 				data-component="geo-most-popular"
 			>
-				<StraightLines
-					cssOverrides={css`
-						display: block;
-					`}
-					count={4}
-					color={themePalette('--straight-lines')}
-				/>
-				<h3 css={headingStyles}>Most viewed</h3>
-				<ul data-link-name="Right hand most popular geo GB">
-					{trails.map((trail, index) => (
-						<MostViewedRightItem
-							key={trail.url}
-							trail={trail}
-							mostViewedItemIndex={index}
+				{testVariant !== 'deeply-read-only' && (
+					<>
+						<StraightLines
+							cssOverrides={css`
+								display: block;
+							`}
+							count={4}
+							color={themePalette('--straight-lines')}
+						/>
+						<h3 css={headingStyles}>Most viewed</h3>
+						<ul data-link-name="Right hand most popular geo GB">
+							{mostReadTrails?.map((trail, index) => (
+								<MostViewedRightItem
+									key={trail.url}
+									trail={trail}
+									mostViewedItemIndex={index}
+								/>
+							))}
+						</ul>
+					</>
+				)}
+				{(testVariant === 'deeply-read-only' ||
+					testVariant === 'deeply-read-and-most-viewed') && (
```

Comment:
Should we simply check if `deeplyReadTrails` is defined here?

Reply:
Yeah that makes sense. As you say in another comment, we could also check the shape of the response. Let's pick this up and improve the code when we remove the test and pick a winner